openslides_backend/shared/interfaces/env.py

```diff
@@ -0,0 +1,17 @@
+from typing import Protocol
+
+
+class Env(Protocol):  # pragma: no cover
+    """
+    Interface for the object containing all environment variables and some
+    helper methods.
+    """
+
+    def __getattr__(self, attr: str) -> str:
```

Comment:
minor remark: I don't think this method should be part of the protocol

Reply:
Mypy complains if I remove this ...

Reply:
Mmh ok, then nevermind